operator/config/samples/karmada.yaml

```diff
@@ -31,35 +31,35 @@ spec:
       serviceSubnet: 10.96.0.0/12
     karmadaAggregatedAPIServer:
       imageRepository: docker.io/karmada/karmada-aggregated-apiserver
-      imageTag: v1.8.0
+      imageTag: v1.11.0
```

Comment:
v1.11.0 or v1.11.1? @zhzhuang-zju 
I remember there was a bug in v1.11.0, https://github.com/karmada-io/karmada/pull/5550.

Reply:
v1.11.1

Reply:
@MAVRICK-1 Could you please update it to v1.11.1?

Reply:
Okay 👍

Reply:
@RainbowMango All image tags for Karmada components, etcd, and kube-apiserver to v1.11.1?

Reply:
just karmada components, which imagetag is v1.8.0 before

Reply:
@RainbowMango thanks for pointing this out, and I also update the issue description

Reply:
Done ✅

Reply:
Done